extensions/cl_intel_host_pipe_symbol.asciidoc

```diff
@@ -0,0 +1,226 @@
+= cl_intel_program_scope_host_pipe
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+
+:blank: pass:[ +]
+
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes.  This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
+
+== Name Strings
+
+`cl_intel_program_scope_host_pipe`
+
+== Contact
+
+Zibai Wang, Intel (zibai 'dot' wang 'at' intel 'dot' com)
+
+== Contributors
+
+Bo Lei, Intel +
+Marco Jacques, Intel +
+Robert Ho, Intel +
+Zibai Wang, Intel +
+
+== Notice
+
+Copyright (c) 2023 Intel Corporation.  All rights reserved.
+
+== Status
+
+Final Draft
+
+== Version
+
+Built On: {docdate} +
+
+== Dependencies
+
+This extension is written against the OpenCL API Specification Version 3.0.7.
+
+Program scope pipes cannot be expressed in the OpenCL kernel language.  But, they can be expressed through SPIR-V entry.  These runtime APIs are only relevant for kernels created from SPIR-V.
```

Comment:
Given that these runtime APIs affect kernels created from SPIR-V, should there be changes to the SPIR-V environment specification in this extension?

CC @garveyjoe

Reply:
@bashbaug , thanks for reviewing this PR. Please merge this PR as-is. The relevant SYCL specification extension(header change) will be added in the llvm doc.